pkg/standalone/standalone.go

```diff
@@ -1013,3 +1049,37 @@ func parseVersionFile(fromDir string) (string, string) {
 
 	return versions[daprRuntimeFilePrefix], versions[dashboardFilePrefix]
 }
+
+func getDefaultRegistry() string {
+	imageRegistryName := githubContainerRegistryName
+	if val := os.Getenv("DEFAULT_IMAGE_REGISTRY_MODE"); val != "" {
+		switch strings.ToLower(val) {
+		case githubContainerRegistryName:
+			imageRegistryName = githubContainerRegistryName
+		case dockerContainerRegistryName:
+			imageRegistryName = dockerContainerRegistryName
+		default:
+			print.FailureStatusEvent(os.Stdout,
+				fmt.Sprintf("Environment variable %s can only be set to %s and %s", "DEFAULT_IMAGE_REGISTRY_MODE", "GHCR", "DOCKERHUB"))
+			os.Exit(1)
+		}
+	}
+	return imageRegistryName
+}
+
+func isImageInGHCR(imageName, imageRegistryName string) (string, bool) {
+	if imageRegistryName != "" {
+		return "", !true
```

Comment:
this is `return "", false` right?

Reply:
yes, to make in sync with the variable name from where it is called, I kept like this.